requirements.txt

```diff
@@ -60,18 +64,18 @@ idna==3.3
     # via requests
 itsdangerous==2.0.1
     # via flask
-jaraco.classes==3.2.1
+jaraco-classes==3.2.1
     # via jaraco-collections
-jaraco.collections==3.4.0
+jaraco-collections==3.4.0
     # via cherrypy
-jaraco.context==4.1.1
+jaraco-context==4.1.1
     # via jaraco-text
-jaraco.functools==3.4.0
+jaraco-functools==3.4.0
```

Comment:
This is going to cause issue for people with older versions of `setuptools` - a PR to 'fix' [this is here](https://github.com/Flexget/Flexget/pull/3412).  I don't think we should keep manually reverting it though - perhaps we should just add docs informing people to update their version of `setuptools`?

More info on this issue [here](https://github.com/jazzband/pip-tools/issues/1576).